app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +17,30 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
```

Comment:
The `collapse-on-mobile` class should only be added if there is a parent breadcrumb:

```
<%= "collapse-on-mobile" if collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } %>"
```